backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -978,6 +997,34 @@ describe('UserResolver', () => {
             }),
           )
         })
+
+        it('returns usercontact object', async () => {
```

Comment:
Please remove this whole it block. We do not want to have a query for user contact

Reply:
but then the UserContact-class itself has 0% converage and the whole coverage is not reached.

So other ideas than simply removing a testcase

Reply:
The UserContact class in the graphql model is never used. Please delete it, delete this test and delete the API call.

Reply:
ok, but then we have to remove the user-query from the UserResolver too, because this query will open quite more security-aspects than the userContact-query.

Reply:
outdated